CHANGELOG.md

```diff
@@ -138,7 +159,6 @@
 **Enhancements:**
 
 - Add commands for S3 logging endpoints [\#9](https://github.com/fastly/cli/pull/9)
-- Add useful next step links to compute deploy [\#8](https://github.com/fastly/cli/pull/8)
```

Comment:
looking into why this was included in the diff.

I'm thinking it may be related to - https://github.com/github-changelog-generator/github-changelog-generator/issues/580

```
$ github_changelog_generator --version
Version: 1.15.2
```